elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```
@@ -7,12 +7,19 @@ $variables: (
   horizontal-spacer: calc(#{pfe-var(container-padding)} * 2),
   vertical-spacer: calc(#{pfe-var(container-padding)} * 2),
   BackgroundColor: pfe-color(surface--lightest),
-  BorderColor: #{pfe-color(ui-base)},
-  FontSize: #{pfe-var(font-size--heading--epsilon)},
-  heading: (FontSize: #{pfe-var(font-size--heading--zeta)}),
-  link: (horizontal-spacer: calc(#{pfe-var(content-spacer)} / 3),
-    vertical-spacer: calc(#{pfe-var(content-spacer)} / 6),
-  ));
+  BorderColor: pfe-color(ui-base),
+  FontSize: pfe-var(font-size--heading--epsilon),
+  heading: (
+      FontSize: #{pfe-var(font-size--heading--zeta)}
```

```suggestion
      FontSize: pfe-var(font-size--heading--zeta)
```

Interpolation not necessary here